pkg/standalone/run.go

```diff
@@ -84,15 +84,19 @@ func (meta *DaprMeta) newAppID() string {
 	}
 }
 
-func (config *RunConfig) validateComponentPath() error {
-	_, err := os.Stat(config.ComponentsPath)
+func (config *RunConfig) validateResourcesPath() error {
+	dirPath := config.ResourcesPath
+	if dirPath == "" {
+		dirPath = config.ComponentsPath
+	}
+	_, err := os.Stat(dirPath)
 	if err != nil {
-		return err
+		return fmt.Errorf("error validating resources path %q : %w", dirPath, err)
 	}
-	componentsLoader := components.NewStandaloneComponents(modes.StandaloneConfig{ComponentsPath: config.ComponentsPath})
+	componentsLoader := components.NewStandaloneComponents(modes.StandaloneConfig{ComponentsPath: dirPath})
 	_, err = componentsLoader.LoadComponents()
 	if err != nil {
-		return err
+		return fmt.Errorf("error validating components in resources path %s : %w", dirPath, err)
```

Comment:
Nit: `%q`?

Reply:
done

Reply:
There were some other places that that might also apply, but all nits.